apps/challenges/views.py

```diff
@@ -84,22 +86,27 @@ def challenge_list(request, challenge_host_team_pk):
         return Response(response_data, status=status.HTTP_406_NOT_ACCEPTABLE)
 
     if request.method == 'GET':
-        challenge = Challenge.objects.filter(creator=challenge_host_team, is_disabled=False)
+        challenge = Challenge.objects.filter(
```

Comment:
I think your editor has made a lot of unnecessary changes which are not needed in this PR. I can't review the changes since there is a lot of irrelevant changes. Can you please remove it.